changelogs/fragments/4194-configurable-passwordstore-locking.yml

```diff
@@ -0,0 +1,2 @@
+minor_changes:
+  - passwordstore lookup plugin - add configurable lock and locktimeout options to avoid race conditions in itself and in the `pass` utility it calls (https://github.com/ansible-collections/community.general/pull/4194).
```

Comment:
```suggestion
  - passwordstore lookup plugin - add configurable ``lock`` and ``locktimeout`` options to avoid race conditions in itself and in the ``pass`` utility it calls. By default, the plugin now locks on write operations (https://github.com/ansible-collections/community.general/pull/4194).
```